executor/memtable_reader_test.go

```diff
@@ -1069,120 +1071,161 @@ func (s *testHotRegionsHistoryTableSuite) TearDownSuite(c *C) {
 	s.testInfoschemaTableSuiteBase.TearDownSuite(c)
 }
 
+func (s *testHotRegionsHistoryTableSuite) newTableKeyRange(tableID int64) []string {
+	sk, ek := tablecodec.GetTableHandleKeyRange(tableID)
+	startKey := helper.BytesKeyToHex(codec.EncodeBytes(nil, sk))
+	endKey := helper.BytesKeyToHex(codec.EncodeBytes(nil, ek))
+	return []string{startKey, endKey}
+}
+func (s *testHotRegionsHistoryTableSuite) newIndexKeyRange(tableID, indexID int64) []string {
```

Comment:
ditto

Reply:
Done by export `NewIndexWithKeyRange` and `NewTableWithKeyRange` for test only now.